editor/src/clj/editor/texture_set.clj

```diff
@@ -250,12 +250,15 @@
 
         ; Pivot point comes from the SpriteGeometry, where (0,0) is center of the image and +Y is up.
         [^double image-pivot-x ^double image-pivot-y] (or (:pivot animation-frame) [0.0 0.0])
-        image-pivot-x (* (or (:width animation-frame) 0.0) image-pivot-x)
-        image-pivot-y (* (or (:height animation-frame) 0.0) image-pivot-y)
+        [width height] (if (or (and (= :size-mode-manual size-mode)) (nil? animation-frame)) size [(:width animation-frame) (:height animation-frame)])
```

Comment:
For manual size or if `animation-frame` doesn't exist, it's important to take `width` and `height` from size so position of the sprite to show sprite in the same position runtime shows